src/hyp3_isce2/water_mask.py

```diff
@@ -1,55 +1,71 @@
 """Create and apply a water body mask"""
-import json
-import subprocess
-from pathlib import Path
-from tempfile import TemporaryDirectory
+from os import system
 
-import geopandas as gpd
+import numpy as np
 from osgeo import gdal
-from pyproj import CRS
-from shapely import geometry
-
-from hyp3_isce2.utils import GDALConfigManager
 
 gdal.UseExceptions()
 
+TILE_PATH = '/vsicurl/https://asf-dem-west.s3.amazonaws.com/WATER_MASK/TILES/'
 
-def split_geometry_on_antimeridian(geometry: dict):
-    geometry_as_bytes = json.dumps(geometry).encode()
-    cmd = ['ogr2ogr', '-wrapdateline', '-datelineoffset', '20', '-f', 'GeoJSON', '/vsistdout/', '/vsistdin/']
-    geojson_str = subprocess.run(cmd, input=geometry_as_bytes, stdout=subprocess.PIPE, check=True).stdout
-    return json.loads(geojson_str)['features'][0]['geometry']
 
+def get_corners(filename, tmp_path: str = ''):
```

Comment:
Having `tmp_path` be an empty strings seems like it could cause issues. Could we make it an `Optional[Path]` with a default of `None`?